### DISCUSSION TITLE
nGrinder 동작 문제 (performance test 사라짐, script 클릭 안됨)

### DISCUSSION THREAD

**Discussion by seungdangdang** [naver/ngrinder#1033] [Q&A]
안녕하세요. 2주 만에 nGrinder에 접속했는데 기존의 test 데이터가 사라지고 scirpt가 클릭이 안되는데 
혹시 동일한 증상을 겪으시는 분이 계실까요?

+ 혹시 JDK 21은 지원하지 않을까요?

**Comment by hyeon0208** [comment]
JDK 8, 11버전만 지원하더라구요.

scirpt가 클릭이 안되는 문제는 저와 동일한 상황인지는 모르겠지만
컨트롤러를 실행시킬때 지정한 디렉토리에 권한이 없어서 발생한 문제로 권한을 부여하니 해결됐습니다.
